### PR TITLE
Invoke imageReplacedCallback with delay.

### LINKED ISSUE #644: Sometimes images upload progress never finish
A user reported this:

https://ios.forums.wordpress.org/topic/upload-images-failed?replies=8

After investigating a bit more it looks the event that reports that the image upload is done is getting lost.

### PR DESCRIPTION
**Pull request by SergioEstevao**
Closes #644

This was similar to a issue we where having with video, I applied the same fix here by adding a delay to the call.

@bummytime can you please review?

### CODE REVIEW

File: Assets/ZSSRichTextEditor.js [hunk 1]
```diff
@@ -783,28 +777,30 @@ ZSSEditor.setProgressOnImage = function(imageNodeIdentifier, progress) {
...
-    }
-    // Wrap link around image
-    var linkTag = '<a href="' + imageNode.attr("src") + '"></a>';
-    imageNode.wrap(linkTag);
+    var joinedArguments = ZSSEditor.getJoinedFocusedFieldIdAndCaretArguments();
+    ZSSEditor.callback("callback-input", joinedArguments);
+    // We invoke the sendImageReplacedCallback with a delay to avoid for
+    // it to be ignored by the webview because of the previous callback being done.
+    var thisObj = this;
+    setTimeout(function() { thisObj.sendImageReplacedCallback(imageNodeIdentifier);}, 500);
```
Comment (bummytime) [inline Assets/ZSSRichTextEditor.js]: @SergioEstevao we used the callback through iframe trick to help avoid callbacks getting lost (`WPHybridCallbacker.prototype.callbackThroughIFrame`). Did you see evidence of this occurring?

Comment (SergioEstevao) [inline Assets/ZSSRichTextEditor.js]: If you remember @bummytime we had similar issues with video insertion and upload.I also have a user on the forum complaining about this. To be honest it doesn't happen 100% of times but I've seen it happen and when it does it's the event  not getting called at all in the objc code.

Only when we go iOS 8 only and can use the new WebView we will be 100% sure.

Comment (bummytime) [inline Assets/ZSSRichTextEditor.js]: Sounds good @SergioEstevao .

**Comment by bummytime** [comment]
:shipit:

**Pull request merged**